### PR TITLE
Adding additional information for ADAS systems

### PR DESCRIPTION
**Pull request by erikbosch**

### CODE REVIEW

File: spec/ADAS/ADAS.vspec [hunk 1]
```diff
@@ -200,6 +200,45 @@ ESC.IsEngaged:
   type: sensor
   description: Indicates if ESC is currently regulating vehicle stability. True = Engaged. False = Not Engaged.
 
+ESC.IsCrossWindDetected:
```
Comment (StefanWerder) [inline spec/ADAS/ADAS.vspec]:
```suggestion
ESC.IsStrongCrossWindDetected:
```
as in the description

File: spec/ADAS/ADAS.vspec [hunk 2]
```diff
@@ -200,6 +200,45 @@ ESC.IsEngaged:
...
+  datatype: boolean
+  type: sensor
+  description: Indicates if ESC system is detecting strong cross winds.
+               True = Strong Cross winds detected.
+               
+ESC.RoadFriction:
+  type: branch
+  description: Road friction values reported by ESC system.
+
+ESC.RoadFriction.Median:
```
Comment (StefanWerder) [inline spec/ADAS/ADAS.vspec]: Is that fitting? The median doesn't have to be the most probable value (https://en.wikipedia.org/wiki/Normal_distribution). Would it be more the mode then?

Comment (erikbosch) [reply spec/ADAS/ADAS.vspec]: But "mode" is only for discrete numbers, right? But here we have continuous/float numbers.

I actually do not know what is needed for the use case. Assume we have the 5 calculations below:

`[1, 2, 7, 45, 45]`

Do we then want median (i.e. 7), or mean (i.e. 20), or the most probable value (45)

Comment (StefanWerder) [reply spec/ADAS/ADAS.vspec]: I would probably go with the term "most probable", leaving the exact interpretation and derivation to the implementation of the road friction estimation.

**Comment by erikbosch** [comment]
Promoted to VSS as part of https://github.com/COVESA/vehicle_signal_specification/pull/437

**Pull request closed without merging**